TrappingRainWater/main.h

```diff
@@ -0,0 +1,11 @@
+#ifndef MAIN_H
+#define MAIN_H
+
+#include <vector>
+
+class Solution {
```

Comment:
give descriptive names to classes and methods. 
Comment everything that is not obvious. 
In this case, describe the problem and the algorithm.